src/js/edu-benefits/1995/components/IntroductionPage.jsx

```diff
@@ -69,28 +68,31 @@ class IntroductionPage extends React.Component {
             </li>
           </ol>
         </div>
-        <div className="row progress-box progress-box-schemaform form-progress-buttons schemaform-buttons">
-          <div className="small-6 usa-width-five-twelfths medium-5 columns">
-            <a href="/education/apply-for-education-benefits/">
-              <button className="usa-button-outline">« Back</button>
-            </a>
-          </div>
-          <div className="small-6 usa-width-five-twelfths medium-5 end columns">
-            <ProgressButton
-              onButtonClick={this.goForward}
-              buttonText="Continue"
-              buttonClass="usa-button-primary"
-              afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Please complete the 22-1990 form to apply for education benefits.
```

Comment:
I think this should be "Please complete the 22-1995 form to apply to manage your education benefits." based on the 5495 form language that Peggy gave us.

Reply:
Thanks @jbalboni, changed.